prompt_toolkit/renderer.py

```diff
@@ -684,9 +684,11 @@ def print_formatted_text(
         else:
             output.reset_attributes()
 
+        # Eliminate newlines
```

Comment:
Technically, these are actually the "carriage returns" which we are eliminating. Would you mind rephrasing the comment? Then it's perfect.

Reply:
ok